bindata/manifests/switchdev-config/files/switchdev-configuration-after-nm.sh.yaml

```diff
@@ -25,11 +43,25 @@ contents:
         echo "load VF driver for $pci_addr"
         VfDirs=$(ls /sys/bus/pci/devices/${pci_addr} | grep virtfn)
 
-        # load VF driver
+        # load VF driver and configure vdpa if needed
         for VfDir in $VfDirs
         do
           VfPciAddr=$(basename "$( readlink -f /sys/bus/pci/devices/${pci_addr}/$VfDir )")
           echo $VfPciAddr > /sys/bus/pci/drivers_probe
+
+          # extract VF id from VfDir
```

Comment:
Since this service file will run before the operator has a chance to run. Are we sure we don't need to load the vDPA driver here?

Reply:
I think the vDPA drivers are loaded in the kernel just when the policy is applied, before the node reboots (please check https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/377/files#diff-5e18b1bb6723228bb51dcd9afe17e63609c26b293a04e6f0a44075ef0f5cfcc2 line 87)

Reply:
`utils.LoadKernelModule` essentially calls modprobe [1]. this is not persistent after reboot AFAIK

so after rebooting a node virtio_vdpa driver would not be loaded when this bit runs. do you need it to perform any of the below operations ?

[1] https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/bindata/scripts/load-kmod.sh

Reply:
No, I don't need the virtio_vdpa driver for the operations below, the command `vdpa dev add name vdpa:0000:65:00.3 mgmtdev pci/0000:65:00.3` for example, can be executed with no issues. Later on the driver is loaded as expected, I tested that after rebooting the node.

Reply:
Only the parent driver is needed at this stage (e.g. mlx5_vdpa)

Reply:
OK cool thx for clarifying it